include/tvm/runtime/threading_backend.h

```diff
@@ -0,0 +1,50 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file threading_backend.h
+ * \brief Utilities for manipulating thread pool threads.
+ */
+#ifndef TVM_RUNTIME_THREADING_BACKEND_H_
+#define TVM_RUNTIME_THREADING_BACKEND_H_
+
+#include <functional>
+#include <memory>
+#include <vector>
+
+namespace tvm {
+namespace runtime {
+namespace threading {
+
+class ThreadGroup {
+ public:
+  class Impl;
+
+   /*!
+    * \brief Starts and manages threads which back a thread pool.
+    *
+    * \param num_workers The number of worker threads to start.
+    * \param worker_callback A callback which is run in its own thread.
+             Receives the worker_id as an argument.
+    * \param include_main_thread Whether to use the main thread as a worker.
```

Comment:
It is more clear to change it to exclude_worker0, We also need to make a precise behavior. For example, if exclude_worker0 is set to true, the callback will only be called with worker_id greater than 0